GitUI/CommandsDialogs/SettingsDialog/Pages/ChecklistSettingsPage.cs

```diff
@@ -27,8 +27,11 @@ public partial class ChecklistSettingsPage : SettingsPageWithHeader
         private readonly TranslationString _gitVersionFound =
             new TranslationString("Git {0} is found on your computer.");
 
-        private readonly TranslationString _unknownSshClient =
-            new TranslationString("Unknown SSH client configured: {0}.");
+        private readonly TranslationString _sshClientNotFound =
+            new TranslationString("SSH client not found: {0}.");
+
+        private readonly TranslationString _otherSshClient =
+            new TranslationString("Other SSH client configured: {0}.");
```

Comment:
Nit:
```suggestion
        private readonly TranslationString _sshClientNotFound = new("SSH client not found: {0}.");

        private readonly TranslationString _otherSshClient = new("Other SSH client configured: {0}.");
```
Should we activate the regarding code analyzer?

Reply:
Added a commit for IDE0090
https://github.com/gitextensions/gitextensions/pull/9095#issuecomment-840428715

This was on my mind, but it would have to be handled when rebasing c#9 new()